Client/Frontend/Browser/Tabs/InactiveTabCell.swift

```diff
@@ -190,13 +190,18 @@ extension InactiveTabCell: UITableViewDataSource, UITableViewDelegate {
         case .inactive, .none:
             guard let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: InactiveTabHeader.cellIdentifier) as? InactiveTabHeader else { return nil }
             headerView.state = hasExpanded ? .down : .right
-            headerView.title = String.TabsTrayInactiveTabsSectionTitle
+            headerView.title = hasExpanded ?
+                .TabsTray.InactiveTabs.TabsTrayInactiveTabsSectionOpenedTitle :
+                .TabsTray.InactiveTabs.TabsTrayInactiveTabsSectionClosedTitle
```

Comment:
Need some conversation to happen with UX for this change, maybe an inbetween solution is to keep the title as is, but read those particular new strings as `.accessibilityLabel` ?

Reply:
Fixed.